pkg/routeagent/controllers/route/route.go

```diff
@@ -256,14 +253,12 @@ func (r *RouteController) handleRemovedEndpoint(obj interface{}) {
 	if object, ok = obj.(*v1.Endpoint); !ok {
 		tombstone, ok := obj.(cache.DeletedFinalStateUnknown)
 		if !ok {
-			utilruntime.HandleError(fmt.Errorf("error decoding object, invalid type"))
-			klog.V(4).Infof("problem decoding object")
+			klog.Errorf("Error decoding object %v, invalid type", obj)
```

Comment:
To be in line with #28 

```suggestion
			klog.Errorf("Couldn't get object from tombstone %#v", obj)
```

Reply:
yeah - we should be consistent. I'll change it.

Reply:
good, thanks!